pkg/controller/migration/convert/convert.go

```diff
@@ -140,6 +140,10 @@ func Convert(ctx context.Context, client client.Client, install *operatorv1.Inst
 		}
 	}
 
+	if err := handleFelixVars(comps); err != nil {
```

Comment:
I think it would be good to have a comment here explaining why this is separate from the other handlers. Maybe something like `Handle the FelixVars as the last handler because we take values and directly load them to the datastore so we want to know that we have migrated all the other config successfully`